onebusaway-android/src/main/java/org/onebusaway/android/util/LocationUtils.java

```diff
@@ -214,6 +219,37 @@ public static GoogleApiClient getGoogleApiClientWithCallbacks(Context context) {
     }
 
     /**
```

Comment:
Could you add the parameters and return values in the Javadocs here?  Easy way to do this is copy the current text to clipboard, remove the comment, and above the method in Android Studio type `/**` and then hit enter.  It should automatically generate the @param and @return template.